nixos/modules/system/activation/activation-script.nix

```
@@ -168,7 +168,7 @@ in
 
       apply = set: {
         script = ''
-          unset PATH
+          export PATH=/empty
```

We would add /empty to the beginning of the path here which I don't think we want to do.

```suggestion
          export PATH=
```

---

Done. The same is already done on line 38 in the same file, not sure why exactly.